### ISSUE TITLE
Reproducing

### ISSUE THREAD

**Issue by chanhoeom** [chufengt/iccv19_attribute#12]
Hello!

First, thanks for releasing a good code for PAR.

I am trying to reproduce the results on the paper!
For a PETA dataset, I download PETA_train_list.txt, PETA_val_list.txt, and PETA_test_list.txt
from here (https://github.com/asc-kit/vespa/tree/master/generated), and train the model using the code!

As a results, I got
![image](https://user-images.githubusercontent.com/25449812/81251283-d9e14980-905d-11ea-8198-bde7b78215eb.png)
which is about 2% lower than the result in the paper.

I have following questions,

1) In the given code, the test accuracy is obtained still using val_loader. Should I change it to test_loader?

2) If it's true, should I also include validation set into a training stage?

3) In addition, according to the paper, attributes that has higher positive ratio that 5% and 1% are used for the evaluation on PETA and RAP, respectively. Is it already reflected in your code? or should I manually remove some attributes lower than the pre-defined ratio?

Thanks in advance :)

**Comment by chanhoeom** [comment]
+ in https://github.com/asc-kit/vespa/tree/master/generated, they made RAP train/val/test lists, I guess, using 52 attributes (c.f. in the paper you said you used 51 attributes).
![image](https://user-images.githubusercontent.com/25449812/81251864-4f99e500-905f-11ea-97e6-9ea946b6637c.png)

I wonder the difference between them..

**Comment by chufengt** [comment]
1. Using test_loader.
2. Yes.
3. We only consider the valid attributes in our codes.
4. For RAP, there should be 51 attributes, but this 'vespa' paper using an additional attribute 'angle' as an auxiliary, so they have 52.

**Comment by chanhoeom** [comment]
Thanks for the reply :)

**Comment by huangyiping-ai** [comment]
Hello, can you share the RAP dataset? Thank you, 18640385797@163.com , I can't apply successfully！
